test/lib/edamam_api_wrapper_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+
+describe EdamamApiWrapper do
+
+  describe "list_recipes method" do
+
+    it 'returns recipes given a keyword' do
+      VCR.use_cassette('list_recipes') do
+        response = EdamamApiWrapper.list_recipes("apple")
+        response.each do |recipe|
+          expect(recipe).must_be_instance_of Recipe
+        end
+      end
+    end
+
+    it 'returns empty array with invalid search' do
+      VCR.use_cassette('invalid_search') do
+        response = EdamamApiWrapper.list_recipes("bbbbbbbb")
+        expect(response).must_equal []
+      end
+    end
+  end
+
+  describe "create_recipe method" do
+    it "creates a valid recipe" do
+      VCR.use_cassette('list_recipes') do
+        response = EdamamApiWrapper.list_recipes("apple")
```

Comment:
I think we don't actually need this test for `EdamamApiWrapper.create_recipe`.

This is because it is a private method, so we can't call it directly. Indeed, the test code is instead calling `list_recipes`, which we know does eventually call `create_recipe`.

Since we can't directly call the method, it's not something we want to have explicit tests for -- this test is basically the same as the one on line 7, so we can remove it.